README.md

```diff
@@ -7,16 +7,18 @@ Spam classification machine learning models for Zenodo records and communities.
 First of all, create a virtualenv, install the depencencies, and run the Jupyter notebook server:
 
 ```bash
-mkvirtualenv --python python3.6 zenodo-classifier
-(zenodo-classifier) pip install -r requirements.txt
-# This will also open Kupyter notebook in your browser
-(zenodo-classifier) jupyter notebook
+# Create a virtual environment
+   mkvirtualenv --python python3.9 zenodo-classifier
+   (zenodo-classifier) pip install -r requirements.txt
+   
+# This will also open Jupyter notebook in your browser
+   (zenodo-classifier) jupyter notebook
 ```
 
 To re-train the model:
 
-1. Run the `dump_zenodo_open_metadata.py` script in production to generate the latest dump
-2. Download the dump locally
+1. Go to Zenodo Open Metadata record `https://doi.org/10.5281/zenodo.787062` to acces all dataset versions
+2. Download the latest dump locally under `datasets`
```

Comment:
```suggestion
2. Download the latest dump locally under `data/`
```

- Slash at the end (to make clear it's a folder)
- I see in the `.gitignore` it's `data/` (unless I'm missing an intermediate step/folder.